Automattic/woocommerce-payments
# Fix/9573 rename payout tos modal

Fixes #9573

#### Changes proposed in this Pull Request
Rename deposit to payout in the TOS disable message.

#### Testing instructions
* On server, remove/edit the row for the account from `wcpay_tos_agreements`
* On client, clear server cache and refresh the Payments screen
* A modal will be shown for accepting the Terms of Service. Click on Decline
* Another modal is displayed with the message 'Your previous transaction and payout data will still be available.' which was earlier 'Your previous transaction and deposit data will still be available.'

-------------------

N/A -

**Post merge**
